examples/python-cicd-with-cloudbuilder/README.md

```diff
@@ -0,0 +1,135 @@
+# Basic Python CI With Cloud Source Repositories and Cloud Build
+
+## Overview
+This repo contains code and instructions for using Google Cloud Source Repositories and Cloud Build to execute unit tests upon code check-in.
+
+The basic steps we need to follow are:
+1. Create a new project in Cloud Source Repository and clone it to your machine
+2. Create your code and unit tests
```

Comment:
It looks like periods are used inconsistently to end bullet points. I'm very pro-period but at the very least they should (or should not) be used consistently for single sentence bullets.

Reply:
Thanks, I went with periods for all bullets.  Should be consistent now.

Reply:
I'm pro-period as well, then there's no ambiguity about if the writer completed their thought or left something off. This is especially important for code comments, I think it's in our style guide. Resolving.